docker/hapi-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       - spring.datasource.driverClassName=org.postgresql.Driver
       - spring.jpa.properties.hibernate.dialect=ca.uhn.fhir.jpa.model.dialect.HapiFhirPostgresDialect
       - hapi.fhir.bulk_export_enabled=true
+      - hapi.fhir.enable_index_missing_fields=true
```

Comment:
Question: Why did you add this? Looking around a little bit, I think it ends up [here](https://github.com/hapifhir/hapi-fhir/blob/7e7aae60606e456e9470f7285889678e39395848/hapi-fhir-jpaserver-model/src/main/java/ca/uhn/fhir/jpa/model/entity/StorageSettings.java#L284) and from the comment it is not obvious to me why you have enabled this. If this is needed, please add a comment line why we are enabling it.

Reply:
I added this change when I faced error while doing [patient bulk export](https://build.fhir.org/ig/HL7/bulk-data/export.html#endpoint---all-patients). I get this error without the configuration changes
 ```
ca.uhn.fhir.batch2.api.JobExecutionFailedException: HAPI-2239:  : HAPI-2223: HAPI-0797: You attempted to start a Patient Bulk Export, but the system has `Index Missing Fields` disabled. It must be enabled for Patient Bulk Export"
```

However, there are no errors for [System Level Export API](https://build.fhir.org/ig/HL7/bulk-data/export.html#endpoint---system-level-export) which is the one we use in our application.

Looks like this is an issue as per this [ticket](https://github.com/hapifhir/hapi-fhir/issues/3127).

I had added this change as I faced this issue when testing manually, I will be removing it as it is not required.

Reply:
Removed this in the latest commit